client/src/js/todos/dragAndDrop.js

```diff
@@ -0,0 +1,154 @@
+import { $, $$, closest } from '../util.js';
+
+export class DragAndDrop {
+  constructor() {
+    this.target = document;
+
+    this.isDragging = null;
+    this.isClone = false;
+
+    this.dragCard = null;
+    this.clonedDragCard = null;
+    this.cardPositionInfo = [];
+    this.moveCardState = false;
+  }
+
+  init() {
+    this.target.addEventListener('mousedown', this.mousedownEventHandler);
+    this.target.addEventListener('mouseup', this.mouseupEventHandler);
+    this.target.addEventListener('mousemove', this.mousemoveEventHandler);
+  }
+
+  mousedownEventHandler = (e) => {
+    if (closest('.default', e.target)) {
+      this.isDragging = true;
+      this.dragCard = closest('.default', e.target);
+      this.column = closest('.column-list', e.target);
+    }
+  };
+
+  mouseupEventHandler = (e) => {
+    this.isDragging = false;
+    this.moveCardState = false;
+    if (this.isClone) {
+      this.clonedDragCard.remove();
+      this.dragCard.classList.replace('place', 'default');
+      this.isClone = null;
+      this.cardPositionInfo = [];
+    }
+  };
+
+  mousemoveEventHandler = (e) => {
+    if (!this.isDragging) return;
+
+    if (!this.isClone) {
+      this.setCardsPosition();
+      this.cloneCard(e);
+    }
+
+    const mouseX = e.clientX;
+    const mouseY = e.clientY;
```

Comment:
```suggestion
    const { clientX: mouseX, clientY: mouseY } = e;
```
destructuring을 활용해 볼 수 있겠네요!